www/pages/error/ErrorPage.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Link from 'next/link';
+
+import styles from './ErrorPage.module.css';
+
+const ErrorPage = ({ err }) => (
+    <div className={ styles.error }>
+        <h1>
+            {err ?
+                <FormattedMessage id="error.internal.title" /> :
+                <FormattedMessage id="error.not_found.title" />
+            }
+        </h1>
+
+        <p>
+            <Link href="/">
+                <a>
+                    <FormattedMessage id="error.return_to_home" />
+                </a>
+            </Link>
+        </p>
+    </div>
+);
+
+ErrorPage.propTypes = {
```

Comment:
Should be required?

Reply:
`err` is only passed by `App` to `ErrorPage` when an internal server error occurs. If `err` is `undefined`, we're dealing with a 404. `err.name`, `err.message`, and `err.statusCode` should be `.required`, despite us not looking using any of them in the example